index.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		commonjs: true,
+		es6: true,
+		node: true,
+	},
+	globals: {
+		t: true,
+		n: true,
+		OC: true,
+		OCA: true,
+		Vue: true,
+		VueRouter: true
```

Comment:
shall we still keep this?

Reply:
Why should we keep `Vue` and `VueRouter`? If you need them, you will have to import them.

Reply:
Well, this is for vue apps, so we need at least vue, no?

Reply:
But if you want to use it, you have to use `import Vue from 'vue'` anyway. So it's a local variable and no global.

Reply:
Ah, yes, I don't know why I needed that in the past